gamestonk_terminal/stocks/screener/finviz_view.py

```diff
@@ -163,15 +167,15 @@ def screener(
         df_screen = df_screen.dropna(axis="columns", how="all")
 
         if sort:
-            if " ".join(sort) in d_cols_to_sort[data_type]:
```

Comment:
@DidierRLopes Here I updated that only one column can be selected for sorting. More than one column we'd lose the auto-complete functionality. And it would require more dev time for it to be compatible with white space in column names.